automatic merge?

es-zenhom opened the pull request.

es-zenhom committed:
Update main.cc

es-zenhom committed:
Update cuts.h

  if (good_jet_p4s.at(vqqjet_idxs.first).pt() > good_jet_p4s.at(vqqjet_idxs.first).pt())
fixing the typo which makes this condition always true

Commit:
Merge branch 'main' of https://github.com/es-zenhom/vbs into main
fixing the typo which makes this condition always true
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~
~

jkguiang commented on analysis/studies/vbsvvhjets/main.cc:
I would prefer not to merge this into the main branch. You can keep this change separate for now.

jkguiang commented on analysis/include/vbsvvhjets/cuts.h:
Sorry in advance for the pedantic review: the comment you changed was indeed wrong, but the change is confusing; I think it should read "Sort the two Vqq jets into leading/trailing".

Otherwise, good catch! Glad you are looking at the code closely.

es-zenhom replied:
yeah sorry for the confusing comment.

es-zenhom committed:
Add files via upload

es-zenhom committed:
Create vbsvvhjets_semimerged

es-zenhom committed:
Delete vbsvvhjets_semimerged

es-zenhom committed:
Delete main.cc

es-zenhom committed:
Create main.cc

es-zenhom committed:
Add files via upload

Commit:
First commit

Commit:
Resolved merge conflicts

Commit:
Description of the changes you made

Commit:
29 aug-gen-level-decays-of wwh

es-zenhom committed:
Create main.cc

This code analyze vvh semi merged channel on gen level. H-bb and H-non bb branches were created. ld and tr branches were created for the bb and non_bb. for the v bosons: Branches for the leading and trailing V were created and then branches for the leading and trailing fermions, coming from each V, were created. The two vbf quarks were identified (ld and tr branches). At the end branches like gen_ld_v_mass were created to get the compined kinamatics of the V,and H bossons decay prodcutcs.

es-zenhom committed:
Update cuts.h

Committed out if (jet_idx == ld_vbsjet_idx || jet_idx == tr_vbsjet_idx) { continue; }
as i am going to choose the vqq jets first and then the vbf jets

es-zenhom committed:
Update cuts.h

added the lines:
  // Skip Vqq jets candidates
            if (jet_i == ld_vqqjet_idx || jet_i == tr_vqqjet_idx) { continue; }
as i chose the vqq jets first, then i need to exclude them while choosing the vbf quarks.
Note: I did this instead of the opposite since i found out that it leads to get results for identifying the vqq jets

es-zenhom committed:
Update collections.h

selecting the vqq jets first then the vbf jets

es-zenhom committed:
Update cuts.h

        // save vbf jet globals to be used in vbs part

es-zenhom committed:
Update cuts.h

        // getting the vqq globals to use it to skip vqq jets candidates
        int ld_vqqjet_idx = globals.getVal<int>("ld_vqqjet_idx");
        int tr_vqqjet_idx = globals.getVal<int>("tr_vqqjet_idx");

es-zenhom committed:
Update collections.h

        // vvhqq globals
        cutflow.globals.newVar<int>("ld_vqqjet_idx");
        cutflow.globals.newVar<int>("tr_vqqjet_idx");
